experimental/generation/generator/templates/professional/library-ChooseIntent.dialog.lg

```diff
@@ -0,0 +1,59 @@
+# template
+- ```
+{
+    "$schema": "${appSchema}",
+    "$kind": "Microsoft.OnChooseIntent",
+    "actions": [
+        {
+            "$kind": "Microsoft.SetProperty",
+            "property": "dialog.recognized",
+            "value": "=turn.recognized"
+        },
+        {
+            "$kind": "Microsoft.ChoiceInput",
```

Comment:
>"$kind": "Microsoft.ChoiceInput", [](start = 12, length = 33)

This really should be an ask--we want open-ended prompts.  Replacing it is more complex, so we can leave it for now but file a bug to come back and fix it.